images/rootfs-mini.yml.in.patch

```diff
@@ -18,68 +18,70 @@
    - GPTTOOLS_TAG
 -  - KEXEC_TAG
  onboot:
--   - name: rngd
--     image: RNGD_TAG
--     command: ["/sbin/rngd", "-1"]
--   - name: sysctl
--     image: linuxkit/sysctl:v0.5
--     binds:
--        - /etc/sysctl.d:/etc/sysctl.d
--     capabilities:
--        - CAP_SYS_ADMIN
--        - CAP_NET_ADMIN
--   - name: modprobe
--     image: linuxkit/modprobe:v0.5
--     command: ["/bin/sh", "-c", "modprobe -a nct6775 w83627hf_wdt hpwdt wlcore_sdio wl18xx br_netfilter dwc3 rk808 rk808-regulator smsc75xx cp210x nicvf tpm_tis_spi rtc_rx8010 gpio_pca953x leds_siemens_ipc127 upboard-fpga pinctrl-upboard leds-upboard xhci_tegra 2>/dev/null || :"]
-    # If you change the order of storage-init don't forget to
-    # change 003-installer in pkg/mkimage-raw-efi accordingly:
-    # storage-init must follow installer
-    - name: storage-init
-      image: STORAGE_INIT_TAG
--   # kdump goes strictly after storage-init
--   - name: kdump
--     image: KDUMP_TAG
+-  - name: rngd
```

Comment:
Curious are all these changes because of the spaces vs tabs? Difficult to understand what was actually added. Mike, could you please "untabify" the newly generated files if it is possible.